src/main/java/org/veupathdb/service/eda/data/core/AbstractPlugin.java

```diff
@@ -1009,4 +1017,22 @@ public List<DynamicDataSpec> getDynamicDataSpecsWithStudyDependentVocabs(String
   public List<DynamicDataSpec> getDynamicDataSpecsWithStudyDependentVocabs(String entityId) {
     return getDynamicDataSpecsWithStudyDependentVocabs(entityId, new ArrayList<>());
   }
-}
+
+  // reverseMatch is for when we want to find entities that are not the specified entity
```

Comment:
`inverseMatch` might be less confusing. Even better maybe a `filterVarSpecsByNotEntityId` wrapper?